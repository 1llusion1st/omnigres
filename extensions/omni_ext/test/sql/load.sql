SELECT omni_ext.load('omni_ext_test');