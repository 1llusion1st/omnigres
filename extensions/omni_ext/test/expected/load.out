SELECT omni_ext.load('omni_ext_test');
 load 
------
 0.1
(1 row)

